website/docs/r/google_project_default_service_accounts.html.markdown

```diff
@@ -0,0 +1,67 @@
+---
+subcategory: "Cloud Platform"
+layout: "google"
+page_title: "Google: google_project_default_service_accounts"
+sidebar_current: "docs-google-project-default-service-accounts-x"
+description: |-
+  Allows management of Google Cloud Platform project default service accounts.
+---
+
+# google_project_default_service_accounts
+
+Allows management of Google Cloud Platform project default service accounts.
+
+When certain service APIs are enabled, Google Cloud Platform automatically creates service accounts to help get started, but
+this is not recommended for production environments as per [Google's documentation](https://cloud.google.com/iam/docs/service-accounts#default).
+See the [Organization documentation](https://cloud.google.com/resource-manager/docs/quickstarts) for more details.
+
```

Comment:
```suggestion
~> This resource works on a best-effort basis, as no API formally describes the default service accounts. If the default service accounts change their name/email or additional service accounts are added, this resource will need to be updated.

```

Let's add a disclaimer so that it's clear what our contract is.